Assets/Project/Scripts/GamePlayScene/BoardManager.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System.Collections.Generic;
+using Project.Scripts.GamePlayScene.Panel;
+using Project.Scripts.GamePlayScene.Tile;
+using Project.Scripts.Utils.Definitions;
+using Project.Scripts.Utils.Library.Extension;
+using UnityEngine;
+
+namespace Project.Scripts.GamePlayScene
+{
+
+    public static class BoardManager
+    {
+        /// <summary>
+        /// タイル、パネルとそれぞれのワールド座標を保持する「ボード」
+        /// </summary>
+        private static readonly Square[,] _board = new Square[StageSize.ROW, StageSize.COLUMN];
+
+        /// <summary>
+        /// パネルの現在位置を保存するパネルから参照できる辞書
+        /// </summary>
+        /// <typeparam name="GameObject">パネルのゲームオブジェクト</typeparam>
+        /// <typeparam name="Vector2">現在位置`(x, y)=>（row, column）`</typeparam>
+        private static readonly Dictionary<GameObject, Vector2> _panelPositions = new Dictionary<GameObject, Vector2>();
```

Comment:
```suggestion
        private static readonly Dictionary<GameObject, Vector2Int> _panelPositions = new Dictionary<GameObject, Vector2Int>();
```
でどうですか。
`(int)`とかなくなる。

Reply:
これ良いコメントだわ